docs/v1/P4Runtime-Spec.mdk

```diff
@@ -618,17 +618,18 @@ To support multiple controllers, P4Runtime uses the same streaming channel
 (available via `StreamChannel` RPC) for session management. The workflow is
 described as follows:
 
-* Each controller is assigned a `role_id` and an `election_id`. The `role_id`
-  defines the role (or realm) that the controller is part of. The `election_id`
-  is unique per role and identifies the master for a specific role. At any point
-  in time, the controller with the largest `election_id` for each role is the
-  master and the rest are slaves. Note that the device does not assign a
+* For each device, every controller is assigned a `role_id` and an
```

Comment:
I'm not sure this adds clarity. I'd rather stick to one controller => one device id & role id (see my comment there: https://github.com/p4lang/p4runtime/issues/149#issuecomment-451687969)